tests/test-util.js

```diff
@@ -201,5 +201,57 @@ require(["jquery", "util"], function ($, util) {
             deepEqual(split.numeric, ["1.0", "0.0", "-3.0"]);
             deepEqual(split.nonNumeric, ["0.0.0", "boaty"]);
         });
+
+        test("Test keepUniqueKeys without removeAll", function() {
+            var keys  = {
+                "a" : new Set([1,2,3,4]),
+                "b" : new Set([3,4,5,6]),
+                "c" : new Set([1,3,4,5,7]),
```

Comment:
```suggestion
                "a" : new Set([1, 2, 3, 4]),
                "b" : new Set([3, 4, 5, 6]),
                "c" : new Set([1, 3, 4, 5, 7]),
```
(just changing the spacing to be consistent)